cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -159,11 +158,11 @@ func Run(s *options.CMServer) error {
 
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartLogging(glog.Infof)
-	eventBroadcaster.StartRecordingToSink(kubeClient.Events(""))
+	eventBroadcaster.StartRecordingToSink(&unversionedcore.EventSinkImpl{Interface: kubeClient.Core().Events("")})
```

Comment:
Argh, `kubeClient.Core().Events("")` almost conforms to `pkg/client/record.EventSink` except for `Patch` :-(